public/app/features/dashboard/components/PubdashFooter/PubdashFooter.tsx

```diff
@@ -4,20 +4,40 @@ import React from 'react';
 import { GrafanaTheme2, colorManipulator } from '@grafana/data';
 import { useStyles2 } from '@grafana/ui';
 
+export interface PublicDashboardFooter {
+  disable: boolean;
+  text: string;
+  logo: string;
+  link: string;
+}
+
 export const PubdashFooter = function () {
   const styles = useStyles2(getStyles);
+  const conf = getPubdashFooterConfig();
 
-  return (
+  return conf.disable ? null : (
     <div className={styles.footer}>
       <span className={styles.logoText}>
-        <a href="https://grafana.com/" target="_blank" rel="noreferrer noopener">
-          powered by Grafana <img className={styles.logoImg} alt="" src="public/img/grafana_icon.svg"></img>
+        <a href={conf.link} target="_blank" rel="noreferrer noopener">
+          {conf.text} <img className={styles.logoImg} alt="" src={conf.logo}></img>
```

Comment:
add alt attribute

Reply:
is url a grafana url? you should use LinkButton or Link in order to use react-router-dom linking

Reply:
This could be any URL, as the user could change it with Whitelabeling

Reply:
great, so it's good in this way